packages/beacon-node/src/network/core/networkCoreWorkerHandler.ts

```diff
@@ -133,7 +133,7 @@ export class WorkerNetworkCore implements INetworkCore {
        * showed that there is a pretty big window of "correct" values but we can always tune as
        * necessary
        */
-      resourceLimits: {maxYoungGenerationSizeMb: opts.maxYoungGenerationSizeMb},
```

Comment:
need `IS_BUN` to decide this parameter or not? also I'm curious if we really need it considering we don't care it for a while @matthewkeil

Reply:
Ah yes, I didn't mean to include this line in this PR. It should be removed (for a future PR) or just set using IS_BUN